akri/Setup-Akri.ps1

```diff
@@ -0,0 +1,210 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+<#
+    .PARAMETER installAkri
+        installAkri boolean flag
+    
+    .PARAMETER uninstallAkri
+        uninstallAkri boolean flag
+
+    .PARAMETER kubeConfigFile
+        clusterName Path
+
+    .PARAMETER namespace
+        namespace
+#>
+param (
+    [Parameter()]
+    [bool] $installAkri,
+
+    [Parameter()]
+    [bool] $uninstallAkri,
+
+    [Parameter(Mandatory = $true)]
+    [string] $kubeconfigFile,
+        
+    [Parameter(Mandatory = $true)]
+    [string] $namespace
+)
+
+function Install-Akri {
+    <#
+    .DESCRIPTION
+        Helper function to install the Akri addon on the target cluster.
+
+    .PARAMETER kubeConfigFile
+        cluster kubeConfigFile
+
+    .PARAMETER namespace
+        namespace
+    #>
+
+    param (
+        [Parameter(Mandatory = $true)]
+        [String] $kubeConfigFile,
+        
+        [Parameter(Mandatory = $true)]
+        [String] $namespace
+    )
+
+    Write-Host "Installing Akri"
+    Write-Host "Creating namespace $namespace"
+    $namespacecheck = kubectl.exe --kubeconfig=$kubeConfigFile get namespace | findstr $namespace
+    if ($null -eq $namespacecheck) {
+        Write-Host "namespace does not exist, Creating"
+        kubectl.exe --kubeconfig=$kubeConfigFile create namespace $namespace
+    }
+    
+    helm.exe --kubeconfig $kubeConfigFile repo add akri-helm-charts https://deislabs.github.io/akri/
+    helm.exe --kubeconfig $kubeConfigFile repo update 
+   
+    Write-Host "Installing Akri charts"
+    helm.exe --kubeconfig $kubeconfigFile install $global:akriRelName akri-helm-charts/akri --namespace $namespace --debug
+    
+    ## Wait for akri-controller pod to be ready
+    Write-Host "Waiting for Pod 'akri-controller' to be ready."
+    while($true) 
+    {
+        $result = (Execute-KubeCtl -ignoreError -kubeconfig $kubeconfigFile -arguments $("wait --for=condition=Ready --timeout=5m -n $namespace pod -l app=akri-controller"))
+        if ($result -ne $null)
+        {
+            break
+        }
+        Start-Sleep -Seconds 15
+    }
+    Write-Host "Pod 'akri-controller' is ready."
+}
+
+function Execute-KubeCtl

```

Comment:
Consider using Invoke instead of execute as per the approved verbs list of PS: https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands?view=powershell-7